src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public static void doWork() {
+        System.out.println("Truck started its work");
+
+    }
+
+    public static void stopWork() {
+        System.out.println("Truck stopped working ");;
+
+    }
```

Comment:
Similar to the `doWork` method, the `stopWork` method should not be static. It should be an instance method that can be overridden in subclasses if needed.